sonic-net/sonic-buildimage
# [Nokia-7215-A1] Update Nokia-7215-A1 platform

Update Nokia-7215-A1 platform to address UT and OC test failures



#### Why I did it
To achieve 100% pass rate on OC testcases

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Run OC test cases on arm64-nokia_ixs7215_52xb-r0 (Nokia-7215-A1) platform 

#### How to verify it

1. Build a Marvell-arm64 target for Nokia-7215-A1 

2. Run this image on arm64-nokia_ixs7215_52xb-r0 and verify all dockers are up and test basic commands like:
    - show version
    - show platform summary
    - show platform syseeprom
    - show platform fan
    - show platform psustatus
    - show platform firmware status
    - show platform temperature
    - show platform ssdhealth

3. Verify ports are up using "show interface status" command
4. Run unit tests and OC test cases.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
